ddp-workspace/projects/ddp-sdk/src/lib/services/activity/validators/activityDecimalRangeValidationRule.ts

```diff
@@ -0,0 +1,31 @@
+import { ActivityAbstractValidationRule } from './activityAbstractValidationRule';
+import { ActivityQuestionBlock } from '../../../models/activity/activityQuestionBlock';
+import { DecimalHelper } from '../../../utility/decimalHelper';
+
+export class ActivityDecimalRangeValidationRule extends ActivityAbstractValidationRule {
+    constructor(
+        public question: ActivityQuestionBlock<any>,
+        public min: number | null = null,
+        public max: number | null = null) {
+        super(question);
+    }
+
+    public recalculate(): boolean {
+        let valid = true;
+
+        if (DecimalHelper.isDecimalAnswerType(this.question.answer)) {
+            const answer = DecimalHelper.mapDecimalAnswerToNumber(this.question.answer);
+            if (this.min && this.max === null) {
+                valid = answer >= this.min;
+            }
+            if (this.max && this.min === null) {
+                valid = answer <= this.max;
+            }
+            if (this.min && this.max) {
+                valid = answer >= this.min && answer <= this.max;
+            }
+        }
```

Comment:
Cool! 
I'd even replace 
`if (min value is set)
valid = valid && (answer >= this.min)
`
with
`if (min value is set)
valid = answer >= this.min
`
 because **valid** is always **true** there. 

 Fixed in f0376703b3a1f27b4b84377329da579612051ea8